### PR TITLE
SR2020 annoucement for teams

### PR DESCRIPTION
**Pull request by PeterJCLaw**
Fixes https://github.com/srobo/competition-team-minutes/issues/117

### CODE REVIEW

**Comment by PeterJCLaw** [comment]
Eww. The CI failure here is that it doesn't like the footnote style of markdown links -- it doesn't understand that it should ignore those tags and not try to make them spellable.

File: SR2020/2019-08-31-announce-SR2020.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+with the details of your team. All future emails about SR2020 will be sent there.
+
+This year we're hosting three Kickstart events, all on **TODO DATE**:
+
+ * at Thread's offices in London
+ * at the University of Southampton
+ * in Cambridge
+
+We're expecting to run with a similar number of teams to SR2019, so we're sure
+that places will go fast. We expect to confirm places at the end of September,
```
Comment (RealOrangeOne) [inline SR2020/2019-08-31-announce-SR2020.md]: I'd suggest rewording this final sentence a bit. Personally i'd rather it be that we confirm places as we go, but we have a fixed deadline to apply. Thoughts?

File: SR2020/2019-08-31-announce-SR2020.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+so sign up now!
+
+  [I'd like to bring a team to SR2020][signup-form]
+
+We look forward to seeing your teams,
+
+Student Robotics
+
+
+[signup-form]: TODO google form?
```
Comment (RealOrangeOne) [inline SR2020/2019-08-31-announce-SR2020.md]: Here, have a signup form https://forms.gle/mCfqNHrF2JWRboch8

File: SR2020/2019-08-31-announce-SR2020.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+subject: SR2020 Registration and Kickstart Announcement
+---
+
+Hi,
+
+We're pleased to announce that registration for Student Robotics 2020 is now
+open. If you would like to compete, please complete this [application form][signup-form]
+with the details of your team. All future emails about SR2020 will be sent there.
+
+This year we're hosting three Kickstart events, all on **TODO DATE**:
```
Comment (RealOrangeOne) [inline SR2020/2019-08-31-announce-SR2020.md]: Maybe just define a ballpark date if we don't have one by the time this email should go out.

File: SR2020/2019-08-31-announce-SR2020.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+ * at Thread's offices in London
```
Comment (trickeydan) [inline SR2020/2019-08-31-announce-SR2020.md]: It's not clear here that Thread is a company.

File: SR2020/2019-08-31-announce-SR2020.md (hunk 1 above)
Comment (trickeydan) [inline SR2020/2019-08-31-announce-SR2020.md]:
```suggestion
that places will go fast. We expect to confirm places by the end of September,
```

File: SR2020/2019-08-31-announce-SR2020.md (hunk 4 above)
Comment (RealOrangeOne) [reply SR2020/2019-08-31-announce-SR2020.md]: My suggestion would be to simply state it's in London (or more specifically, Whitechapel), rather than at specific offices. Said information is likely more useful to people, anyway.

File: SR2020/2019-08-31-announce-SR2020.md (hunk 1 above)
Comment (PeterJCLaw) [reply SR2020/2019-08-31-announce-SR2020.md]: I'm not sure what the benefit of explicitly confirming as we go is? My assumptions here are:
- we'd confirm as soon as we could anyway, we're limited by confirmation of venue size and complete kits to hand out
- a hard deadline is meaningless as we'd happily accept applications after that if we had space

My aim here is to provide a feeling for when teams should expect to hear back by, with the previous sentence being the motivation for getting on with it.

File: SR2020/2019-08-31-announce-SR2020.md (hunk 2 above)
Comment (PeterJCLaw) [reply SR2020/2019-08-31-announce-SR2020.md]: ta, f0f47f9

File: SR2020/2019-08-31-announce-SR2020.md (hunk 4 above)
Comment (PeterJCLaw) [reply SR2020/2019-08-31-announce-SR2020.md]: Yeah, I've done the same for Southampton so they're all similar: a478e47.

File: SR2020/2019-08-31-announce-SR2020.md (hunk 3 above)
Comment (PeterJCLaw) [reply SR2020/2019-08-31-announce-SR2020.md]: cfbd8fa

File: SR2020/2019-08-31-announce-SR2020.md (hunk 1 above)
Comment (RealOrangeOne) [reply SR2020/2019-08-31-announce-SR2020.md]: Theoretically, but I believe teams generally prefer to have as much notice as possible, as it helps with arranging getting to the kickstart events.

Comment (sedders123) [reply SR2020/2019-08-31-announce-SR2020.md]: In my opinion the current wording gives us the most options. There is no reason we couldn't confirm teams earlier than the end of September but if for some reason we can't confirm as we go we have given teams a deadline that is achievable for us.

**Comment by RealOrangeOne** [comment]
This has now been sent.
